actionpack/lib/action_dispatch/middleware/api_public_exceptions.rb

```diff
@@ -0,0 +1,47 @@
+module ActionDispatch
+  class ApiPublicExceptions
```

Comment:
In fact, ApiPublicExceptions is never used apart from tests (it's not included in the default middleware like rails-api gem does [here](https://github.com/rails-api/rails-api/blob/e8268530f5c395ef17aa30b477bf097080375e3c/lib/rails-api/application.rb#L75)).

After testing for a while, I'm pretty convinced that exceptions are rendered in the json response fine without the need of this module. The existent PublicException is doing exactly the same, I think.

@spastorino I think we can just remove this file and related tests, but maybe I'm missing something?